assessments/guide/security-reviewer.md

```diff
@@ -53,14 +53,37 @@ guidelines for conflict of interest situations are important to prevent:
 - SIG-Security chairs and assessment leads could intentionally or unintentionally limit the participation of an individual unfairly by asserting conflict of interest
 - Security reviews being stalled while groups belabor on who should be allowed to participate
 
-The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts.
-Hard conflicts makes a reviewer ineligible to review a project.
-A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
-A reviewer with a soft conflict is required to document the nature of the conflict in the
-security assessment documentation. It is the responsibility of the security reviewer
-to make known his/her own conflict of interests.
-
-For each project, 2 SIG-Security chairs must sign off on the conflicts presented to them that the assessment lead has no conflicts, and reviewers have no hard conflicts. Reasons for accepting and rejecting conflicts should be documented.
+The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts:
+* Hard conflicts make a reviewer ineligible to review a project.
+* A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
+* Soft conflicts are expected for some reviewers, and therefore are fine as long
+  as the review team includes enough different perspectives to ensure a
+  balanced assessment.
+
+A reviewer with a any conflict or potential conflect is required to document the nature of the conflict when volunteering as a reviewer, or at any time as they
+become aware of a potential conflict. It is the responsibility of the security 
+reviewer to make known his/her own conflict of interests and ensure that
+their declaration has been reviewed and approved by lead security reviewer.
```

Comment:
I think the lead being aware is good, but this does also seem to mildly conflict with the statement below about the facilitator / chairs reviewing the conflict of interest.  Do we want the lead assessor to have the final say here?  I'd prefer it was centralized across reviews.

Reply:
The above section is attempting to make clear that individual reviewers must make sure they document a conflict-of-interest if it happens after initial approval (which happened a while back via acquisition / merger) and its not enough to just comment on the issue, they have to also make sure the lead security reviewer acknowledges their note -- of course that may entail wider review depending on the nature of the conflict.  Not sure how to make that more clear...